tests/test_validation_tool.py

```diff
@@ -267,8 +267,8 @@ def test_validation_original_and_parsed_not_equal() -> None:
     )
     validation_result = _validate_message(
         Path("tests/validation_tool/ethernet/frame/invalid/ethernet_invalid_too_long.raw"),
-        True,
-        ethernet_too_short_value,
+        valid_original_message=True,
+        message_value=ethernet_too_short_value,
```

Comment:
Maybe we should use named parameters consistently here?

Reply:
That would mean that a use of `True` or `False` in a function call always lead to the necessity of naming all arguments. I don't think that would be beneficial.

BTW: This discussed change is already on `main` and we shouldn't do any non SPDM-related changes on this branch. I don't think it makes sense to rebase this feature branch to `main` at all. It certainly makes sense to rebase `spdm` to `main` from time to time, but this feature branch should only be rebased to `spdm`.

Reply:
> That would mean that a use of `True` or `False` in a function call always lead to the necessity of naming all arguments. I don't think that would be beneficial.

Fair enough, I can see scenarios where this would be annoying.

> BTW: This discussed change is already on `main` and we shouldn't do any non SPDM-related changes on this branch. I don't think it makes sense to rebase this feature branch to `main` at all. It certainly makes sense to rebase `spdm` to `main` from time to time, but this feature branch should only be rebased to `spdm`.

You're right. Rebasing onto `main` was not a great idea. I reverted that - we can rebase `spdm` onto `main` once this PR is merged.